### PR TITLE
feat(fliqr-ai): Get Account Details and Get Account Flows actions

### PR DESCRIPTION
**Pull request by drona2938**
ANNOUNCEMENT=true

### CODE REVIEW

File: packages/ui/common/src/lib/environments/environment.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:3000/v1',
+  apiUrl: 'https://legendary-garbanzo-q9995r99p4pfx6px-3000.app.github.dev/v1',
```
Comment (codegino) [inline packages/ui/common/src/lib/environments/environment.ts]: revert?

File: packages/server/api/.env [hunk 2]
```diff
@@ -15,4 +15,4 @@ AP_TELEMETRY_ENABLED=false
 AP_TRIGGER_DEFAULT_POLL_INTERVAL=1
 AP_WEBHOOK_TIMEOUT_SECONDS=30
 AP_PIECES_SYNC_MODE=NONE
-AP_WEBHOOK_URL="http://localhost:3000"
\ No newline at end of file
+AP_WEBHOOK_URL="https://legendary-garbanzo-q9995r99p4pfx6px-3000.app.github.dev"
```
Comment (codegino) [inline packages/server/api/.env]: revert file changes?

Comment (drona2938) [reply packages/server/api/.env]: Yes, What I need to do, could not understand,

I followed the tutorial of pieces development. 

Let me know, what I need to change in them, you can edit from your side, if possible

File: packages/ui/common/src/lib/environments/environment.ts (hunk 1 above)
Comment (drona2938) [reply packages/ui/common/src/lib/environments/environment.ts]: Yes, What I need to do, could not understand,

I followed the tutorial of pieces development. 

Let me know, what I need to change in them, you can edit from your side, if possible

File: packages/pieces/community/fliqr-ai/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  },
+});
+
+export const fliqrAi = createPiece({
+  displayName: 'Fliqr AI',
+  description:
+    'Omnichannel AI chatbot enhancing customer interactions across WhatsApp, Facebook, Instagram, Telegram, and 6 other platforms.',
+
+  minimumSupportedRelease: '0.20.0',
+  logoUrl: 'https://storage.googleapis.com/wtl/w/254/icon3929.png',
```
Comment (codegino) [inline packages/pieces/community/fliqr-ai/src/index.ts]: Activepieces team will create the logo for you. Update the `logoUrl` to this.

```suggestion
  logoUrl: 'https://cdn.activepieces.com/pieces/fliqr-ai.png',
```

File: packages/ui/common/src/lib/environments/environment.ts (hunk 1 above)
Comment (codegino) [reply packages/ui/common/src/lib/environments/environment.ts]: If I'm not mistaken, this file should not be modified when adding a piece. Just remove your changes.

File: packages/server/api/.env (hunk 2 above)
Comment (codegino) [reply packages/server/api/.env]: If I'm not mistaken, this file should not be modified when adding a piece. Just remove your changes.

**Review by drona2938 — COMMENTED** [review]
reverted the changes in env and config file

**Comment by kishanprmr** [comment]
@drona2938 Can you provide API docs for Fliqr AI?

**Comment by drona2938** [comment]
@kishanprmr 

Docs URL : https://fliqrscalar.apidocumentation.com/reference
Dashboard URL : https://app.fliqr.ai
Logo : 
Fliqr Logo

**Comment by abuaboud** [comment]
@allcontributors please add @drona2938 for plugin

**Comment by kishanprmr** [comment]
@drona2938, please create a separate PR for the RazorPay piece. I'm removing it from this PR.

**Comment by drona2938** [comment]
@kishanprmr Okay, I will create a separate PR for Piece RazorPay

**Pull request merged**